src/trs/hmtrs.rs

```diff
@@ -1,24 +1,82 @@
-use super::utils::{log_n_of, logsumexp};
```

Comment:
Yeah, not sure why, but my formatter kept throwing these `use super::*` lines above my module docs.